backend/email_sender_with_restart.py

```
@@ -0,0 +1,259 @@
+                category = email_filter.filter_email(alert.note.lower() if alert.note else "")
+                logger.debug(f"Category result: '{category}' from note: '{alert.note}'")
+                
+                if category != "Uncategorized":
+                    category_map[category].append(alert)
+                    processed_count += 1
+                    
+                    # Mark as sent
+                    alert.sent = True
+                    alert.save()
+                    
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)